app/src/main/java/org/mozilla/fenix/HomeActivity.kt

```diff
@@ -218,17 +222,36 @@ open class HomeActivity : LocaleAwareAppCompatActivity() {
         themeManager.applyStatusBarTheme(this)
     }
 
-    private fun setupToolbarAndNavigation() {
+    fun getSupportActionBarAndInflateIfNecessary(): Toolbar? {
```

Comment:
This should always return a non-null Toolbar, right? Can we update the return type?